docs/source/visual-gen/features/quantized-attention.md

```
@@ -0,0 +1,148 @@
+# VisualGen Quantized Attention
+
+```{note}
+This page is an unindexed draft until the VisualGen documentation hub is introduced.
+```
+
+- [Overview](#overview)
+  - [Recipes](#recipes)
+  - [Configuration Surface](#configuration-surface)
+- [QK16PV8 (CUTEDSL)](#qk16pv8-cutedsl)
+- [SageAttention (TRTLLM)](#sageattention-trtllm)
+- [Block-Scaled Q/K: MXFP8 and NVFP4 (CUTEDSL)](#block-scaled-qk-mxfp8-and-nvfp4-cutedsl)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)